metrics/web/docs/features/deep_links/diagrams/deep_links_integration_component_diagram.puml

```diff
@@ -0,0 +1,35 @@
+@startuml deep_links_integration_component_diagram
```

Comment:
Would it be great to mark new relationships (relationships we are going to implement with this feature) with another color to underline that these relationships are part of the new feature?